src/delfitlm/settings.py

```diff
@@ -210,3 +211,56 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers' : False,
+    'loggers': {
+        'django_logger': {
+            'handlers': ['debug_console', 'debug', 'info', 'warning', 'error'],
+            'level': 1
+        }
+    },
+    'handlers': {
+        'std_err': {
+            'class': 'logging.StreamHandler'
+        },
+        'debug_console': {
+            'class': 'logging.StreamHandler',
+            'level': 'WARNING',
+            'formatter': 'default',
+        },
```

Comment:
This config outputs logs also in docker. I set it to warning, so it will log warnings and errors. Logs generated by `manage.py `commands after `docker exec -it delfitlm_app_1 /bin/bash` would appear in `docker compose logs` though.